client/blocks/reader-post-card/docs/example.jsx

```diff
@@ -41,18 +40,19 @@ const searchItems = [
 	}
 ];
 
-const RefreshCards = React.createClass( {
-	displayName: 'RefreshCard',
+const RefreshCards = () => (
+	<div className="design-assets__group">
+		<div>
+			{ searchItems.map( item =>
```

Comment:
Just curious... would skipping wrapping parens not cause any issue or go against code guidelines here?

I've usually wrapped components like so, if they are not single-liners
```
{ searchItems.map( item => (
    <RefreshPostCard
        ....
    />  
) ) }
```